packages/strapi-plugin-upload/admin/src/components/EditForm/index.js

```diff
@@ -79,7 +79,9 @@ const EditForm = forwardRef(
     useEffect(() => {
       if (isImg || isVideo) {
         if (prefixedFileURL) {
-          setSrc(prefixedFileURL);
+          // Adding performance.now forces the browser no to cache the img
+          // https://stackoverflow.com/questions/126772/how-to-force-a-web-browser-not-to-cache-images
+          setSrc(`${prefixedFileURL}?${performance.now()}`);
```

Comment:
I don't think we need to add this here as the file will be fetched on every re reader furthermore since the file is fetched when the modal is opened there's no cache. Can you explain why you added it?